test/extended/util/cli.go

```diff
@@ -527,6 +527,22 @@ func (c *CLI) Run(commands ...string) *CLI {
 	return nc.setOutput(c.stdout)
 }
 
+// Run command without extra args
+func (c *CLI) RunWithoutGlobalArgs(commands ...string) *CLI {
```

Comment:
did you have to introduce this because Run appends --config and --namespace after the other args?  If so, why not fix Run to append commands after the --config and --namespace instead of introducing a new function?

Reply:
The `Run` function will add the `--config=xxx`/`--namespace=xxx` on the end of the command. But, it inappropriate for some commands, for example, `oc exec -n openshift-apiserver-operator openshift-apiserver-operator-59487cc4bf-7ws9q -c openshift-apiserver-operator -- cat /etc/redhat-release`. 
And, I also thought to change the `Run` function, but as you know, Golang doesn't support the default arguments. I worried the changes will impact other old invoking. So, I create a new one.
Or we can change the `Run` function to add `--config=xxx`/`--namespace=xxx` args at the begining of the commands? What do you suggest?

Reply:
i would expect that adding it to the beginning should be fine and if it breaks a test it should be caught in CI as part of your PR.

Reply:
Ok, thanks! I will modify the `Run` function instead of creating a new one.

Reply:
@jianzhangbjz some ubi repo are enabled by default.
sh-4.2$ yum repolist
repo id                                                                   repo name                                                                                                                          status
ubi-7/x86_64                                                              Red Hat Universal Base Image 7 Server (RPMs)                                                                                       832
ubi-7-optional/x86_64                                                     Red Hat Universal Base Image 7 Server - Optional (RPMs)                                                                             18
ubi-7-rhah/x86_64                                                         Red Hat Universal Base Image Atomic Host (RPMs)                                                                                      3
ubi-7-rhscl/x86_64                                                        Red Hat Software Collections for Red Hat Universal Base Image 7 Server (RPMs)

Reply:
Thanks for your information! @anpingli Removed the `RunWithoutGlobalArgs` function. @bparees